src/api.cpp

```diff
@@ -98,7 +101,7 @@ void destroy_stripes(vector<double*> &dm_stripes, vector<double*> &dm_stripes_to
 }
 
 
-void initialize_mat(mat_t* &result, biom &table, bool is_upper_triangle) {
+void initialize_mat(mat_t* &result, biom_interface &table, bool is_upper_triangle) {
```

Comment:
I could not get the code to compile using `biom` rather than `biom_interface`, but I confess I'm not fully sure why